content/docs/components/central-dash/overview.md

```diff
@@ -1,10 +1,11 @@
 +++
 title = "Central Dashboard"
-description = "All about central dashboard in the Kubeflow Ecosystem"
+description = "An overview of the Kubeflow user interface (UI)"
 weight = 10
 +++
 
 Your Kubeflow deployment includes Central Dashboard as a key component to provide quick access to all components deployed in your cluster. Some features include:
+
 - Quick action, pipelines cards, metrics, and high level kubernetes events for an overview of your jobs and cluster in one view
 - Housing view for all sub services running inside the cluster: `Pipelines`, `Katib`, `Notebooks`, etc.
 - Helps users ramp up their environment with a [registration flow](/docs/central-dash/registration-flow/) (for multi-user isolation, or the first namespace for a shared deployment)
```

Comment:
Both the links in the last two points are broken.

Reply:
Fixed!